File: src/main/js/firma-evenimente/src/app/Models/home/add-eveniment.ts
import { MomenteArtisticeEveniment } from './momente-artistice-eveniment';
import { ConfigurariTipAccesEveniment } from 'src/app/Models/home/configurari-tip-acces-eveniment';
import { TipMeniuEveniment } from './tip-meniu-eveniment';
import { MediaEveniment } from './media-eveniment';
import { ElementeDesignEveniment } from './elemente-design-eveniment';
import { BauturaEveniment } from './bautura-eveniment';
export class AddEveniment {
  id_utilizator: number;
  id_sub_eveniment: number;
  id_locatie: number;
  data_inceput_eveniment: Date;
  data_sfarsit_eveniment: Date;
  mentiuni: string;
  bauturaEveniment: BauturaEveniment[];
  elementeDesignEveniment: ElementeDesignEveniment[];
  momenteArtisticeEveniment: MomenteArtisticeEveniment[];
  mediaEveniment: MediaEveniment[];
  tipMeniuEveniment: TipMeniuEveniment[];
  configurariTipAccesEveniment: ConfigurariTipAccesEveniment[];
}

File: src/main/java/com/firmaevenimente/firmaevenimente/models/wizard/AddEveniment.java
package com.firmaevenimente.firmaevenimente.models.wizard;

import java.util.ArrayList;
import java.util.Date;

public class AddEveniment {
    public int id_eveniment;
    public int id_utilizator;
    public int id_sub_eveniment;
    public int id_locatie;
    public Date data_inceput_eveniment;
    public Date data_sfarsit_eveniment;
    public String mentiuni;
    public ArrayList<BauturaEveniment> bauturaEveniment;
    public ArrayList<ElementeDesignEveniment> elementeDesignEveniment;
    public ArrayList<MomenteArtisticeEveniment> momenteArtisticeEveniment;
    public ArrayList<MediaEveniment> mediaEveniment;
    public ArrayList<TipMeniuEveniment> tipMeniuEveniment;
    public ArrayList<ConfigurariTipAccesEveniment> configurariTipAccesEveniment;
}
